test_newt_cmds.sh

```diff
@@ -24,17 +24,12 @@ declare -a commands=(
 "newt version"
 "newt new project"
 "cd project/"
-"newt install"
+"newt install -v --escape=false apache-mynewt-core apache-mynewt-nimble"
 "newt info"
-"newt build my_blinky_sim"
-"newt clean my_blinky_sim"
-"newt build all"
-"newt test @apache-mynewt-core/kernel/os"
```

Comment:
Were the lines above removed on purpose? (or should they only be skipped on Windows?)

Reply:
I removed those to simplify the setup for the OSes but now that I think about it we can install both arm toolchain and gcc when running TEST_NEWT.